### ISSUE TITLE
ESP8266 Dies

### ISSUE THREAD

**Issue by jhagberg** [marcoschwartz/aREST_UI#28]
Hej.

It looks like the aREST_UI is hanging after some min using the UI version. 

It responds to ping but nothing else. I tried the plain aREST and got the same problem at first. Checking out the https://github.com/marcoschwartz/aREST/pull/183 it was more stable. 
Any tips on how to debug this?

**Comment by jhagberg** [comment]
Just tried this on a Adafruit HuZZAH 4M and I managed to get these printouts before it dies. 

Live heap is: 38224.00
Live heap is: 38224.00
Live heap is: 27640.00
Live heap is: 4840.00

DEAD

**Comment by marcoschwartz** [comment]
Hi, unfortunately it is an issue with the HTTP module on the ESP8266 Arduino library - for now I was not able to find a fix despite a lot of messages I posted on the library's GitHub repository

**Comment by jhagberg** [comment]
Yes, I failed also to get a fix. I am now using the ESP8266Webserver lib and hardcoding the html in progmem. So far stable. Thanks for your response.
